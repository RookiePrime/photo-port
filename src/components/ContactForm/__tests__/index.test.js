import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ContactForm from '..';

afterEach(cleanup);

const categories = [
    { name: 'portraits', description: 'Portraits of people in my life' }
]

describe('ContactForm is rendering', () => {

    it('renders', () => {
        render(<ContactForm></ContactForm>)
    });

    it('matchess snapshot', () => {
        const { asFragment } = render(<ContactForm></ContactForm>);
        expect(asFragment).toMatchSnapshot();
    });
});

describe('Stuff is visible', () => {
    it('renders the h1', () => {
        const { getByTestId } = render(<ContactForm></ContactForm>);
        expect(getByTestId('Contact me')).toHaveTextContent('Contact me');
    });
    it('renders the submit button', () => {
        const { getByTestId } = render(<ContactForm></ContactForm>);
        expect(getByTestId('Submit')).toHaveTextContent('Submit');
    });
});