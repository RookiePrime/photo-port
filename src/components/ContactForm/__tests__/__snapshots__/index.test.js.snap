// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm is rendering matchess snapshot 1`] = `[Function]`;
